desktop/src/app/components/navbar/warnings/modals/fix-outliers-modal.component.ts

```diff
@@ -70,6 +74,7 @@ export class FixOutliersModalComponent {
             for (let fieldName of Object.keys(document.warnings.outliers.fields)) {
                 const field: Field = CategoryForm.getField(category, fieldName);
                 if (!this.hasOutlierValue(document, field)) continue;
+                if (this.inputType == 'checkboxes' && field.inputType != 'checkboxes') continue;
```

Comment:
Code style: === and !== should be used per default (also below and in html file).

Also, the "replace all" option label should maybe be adjusted, as this option now doesn't replace all values anymore like it says but only all values of fields with a matching input type (checkbox or non-checkbox). (It's hard to come up with something that doesn't sound super complicated though...)